sky/templates/runpod-ray.yml.j2

```diff
@@ -0,0 +1,108 @@
+cluster_name: {{cluster_name_on_cloud}}
+
+# The maximum number of workers nodes to launch in addition to the head node.
+max_workers: {{num_nodes - 1}}
+upscaling_speed: {{num_nodes - 1}}
+idle_timeout_minutes: 60
+
+provider:
+  type: external
+  module: sky.skylet.providers.runpod.RunPodNodeProvider
```

Comment:
nto very familiar with new provisioner API; is this still needed? I don't find the file in the PR

Reply:
Good catch! we should use `sky.provision.runpod` here.Updated. Thanks!